frontend/tet/admin/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@frontend/tet-admin",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "12.0.7",
```

Comment:
next 12 wont work with the current custom babel config. You probably have to use the latest next 11 version.